contracts/0.8.25/vaults/LazyOracle.sol

```diff
@@ -238,11 +246,13 @@ contract LazyOracle is ILazyOracle, AccessControlEnumerableUpgradeable {
     /// @notice update the sanity parameters
     /// @param _quarantinePeriod the quarantine period
     /// @param _maxRewardRatioBP the max EL CL rewards
+    /// @param _maxLidoFeeRatePerSecond the max Lido fee rate per second
     function updateSanityParams(
```

Comment:
prob can use `uint256` for every param here to avoid ABI encoder mysteries during debugging